pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -20,13 +22,20 @@ import (
 
 const (
 	// DIDExchange did exchange protocol
-	DIDExchange        = "didexchange"
+	DIDExchange = "didexchange"
+
+	//Inbound message type
 	connectionSpec     = metadata.AriesCommunityDID + ";spec/connections/1.0/"
 	connectionInvite   = connectionSpec + "invitation"
 	connectionRequest  = connectionSpec + "request"
 	connectionResponse = connectionSpec + "response"
+
+	//ControllerReceiveInvitation Controller message types for receive invitation
+	ControllerReceiveInvitation = "receive-invitation"
```

Comment:
this msgType is not part of the did-exchange protocol. Invitations are already identified with the `connectionInvite` property.

Reply:
done